src/bp/core/repositories/telemetry_payload.ts

```diff
@@ -0,0 +1,132 @@
+import { ConfigProvider } from 'core/config/config-loader'
+import { DataRetentionService } from 'core/services/retention/service'
+import { inject, injectable } from 'inversify'
+import Knex from 'knex'
+import _ from 'lodash'
+import moment from 'moment'
+
+import Database from '../database'
+import { TYPES } from '../types'
+
+const DEFAULT_ENTRIES_LIMIT = 1000
+
+type TelemetryEntries = {
+  url: string
+  events: Array<JSON>
+}
+
+type TelemetryEntry = {
+  uuid: string
+  payload: JSON
+  available: Knex.Bool
+  lastChanged: Knex.Date
+  creationDate: Knex.Date
+}
```

Comment:
Please use interfaces unless you really need to use type. 

Also, use the native types instead of Knex internal types. This is the object that you move around on the backend, so we use pure JS there. JSON = any, Knex.Bool = boolean, Knex.Date = Date

Reply:
fixed in c69de5cf271e960e8deebbcc1c7b1abd0f3af91c